README.md

```diff
@@ -468,16 +468,22 @@ The `dtsSelectItem` directive is used to mark DOM elements as selectable items.
 
 **Inputs**
 
-| Input         | Type | Default            | Description                                  |
-| ------------- | ---- | ------------------ | -------------------------------------------- |
-| dtsSelectItem | any  | Directive Instance | Value that is used when the item is selected |
+| Input         | Type     | Default            | Description                                  |
+| ------------- | -------- | ------------------ | -------------------------------------------- |
+| dtsSelectItem | any      | Directive Instance | Value that is used when the item is selected |
+| dtsDisabled   | boolean  | false              | Disable selection for item                   |
```

Comment:
I'd like to change the description to something like `Whether the item is selectable or not`.